config/v1/types_authentication.go

```diff
@@ -192,6 +192,14 @@ type OIDCProvider struct {
 	// +required
 	Issuer TokenIssuer `json:"issuer"`
 
+	// OIDCClients contains configuration for the platform's clients that
+	// need to request tokens from the issuer
+	//
+	// +listType=map
+	// +listMapKey=component.name
```

Comment:
name and namespace, right?  I think you just want Component?  Does that not work?

Reply:
the docs say mapKey must be of scalar type